### PR TITLE
ROC-3304: Defence PDF content revision

### PR DESCRIPTION
**Pull request by singh-surjeet**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-3304

### Change description ###
Using single defendant response PDF across all user claimant/defendant/staff. No seal, renamed and most of content revised. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/party/Organisation.java [hunk 1]
```diff
@@ -35,6 +35,10 @@ public Organisation(
         return Optional.ofNullable(companiesHouseNumber);
     }
 
+    public Optional<String> getBusinessName() {
+        return Optional.ofNullable(getName());
```
Comment (timja-kainos) [inline domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/party/Organisation.java]: what's this for? name is mandatory and already exists

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html [hunk 2]
```diff
@@ -62,6 +85,7 @@
     table td:first-child {
       width: 220px;
       vertical-align: top;
+      /*font-weight: 700;*/
```
Comment (timja-kainos) [inline src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: no leaving commented out code in please

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html [hunk 3]
```diff
@@ -83,124 +111,95 @@
...
-      {% endif %}
-      {% if claimant.businessName is defined %}
-        <tr>
-          <td>Business name:</td>
-          <td>
-            {{ claimant.businessName }}
-          </td>
-        </tr>
       {% endif %}
+      {% if defendant.businessName is defined%}
```
Comment (timja-kainos) [inline src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: spacing after defined please

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html [hunk 4]
```diff
@@ -291,190 +290,109 @@
...
-  </div>
-  <div class="section-body">
-    <table>
+      {% if defendant.events is defined %}
       <tr>
-        <td>Claim amount:</td>
-        <td>
-          {{ claim.claimAmount }}
+        <td colspan="2">
+          <strong>Defendant's timeline of what happened:</strong>
```
Comment (timja-kainos) [inline src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: use an apostrophe not a quote i.e. `’`

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html [hunk 5]
```diff
@@ -291,190 +290,109 @@
...
       </tr>
+      {% endfor %}
       {% endif %}
+      {% if defendant.evidences is defined %}
       <tr>
-        <td>Accrued interest:</td>
-        <td>
-          Interest will accrue at the daily rate of {{ claim.interest.dailyAmount }} up to the date of judgment.
+        <td colspan="2">
+          <strong>Defendant's evidence:</strong>
```
Comment (timja-kainos) [inline src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: same here ’

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html [hunk 6]
```diff
@@ -291,190 +290,109 @@
...
+      <td><strong>Settle the claim</strong></td>
+      <td><p>
+        Unless you’ve been asked not to, you can still contact defendant directly.
+        If you can reach an agreement you may not have to go to a hearing.
+      </p></td>
+    </tr>
+    <tr>
+      <td><strong>Mediation</strong></td>
+      <td>
+        <p>If the defendant has asked for mediation then we'll contact you to ask if you want to mediate.</p>
```
Comment (timja-kainos) [inline src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: same here ’

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html [hunk 7]
```diff
@@ -291,190 +290,109 @@
...
+        <p>If the defendant doesn't want to mediate then the case will be reviewed by a judge and might go to a hearing.
```
Comment (timja-kainos) [inline src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: and here ’

**Comment by singh-surjeet** [comment]
Review comments and changes to AC addressed.

**Comment by sabahirfan** [comment]
Please resolve conflicts.

**Comment by timja-kainos** [comment]
Tested by Clem on demo

**Pull request merged**